2022-03-02 07:49:20,407 INFO    MainThread:18492 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/.config/wandb/settings
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/settings
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_init.py:_log_setup():386] Logging user logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220302_074920-10hbgbtf/logs/debug.log
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_init.py:_log_setup():387] Logging internal logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220302_074920-10hbgbtf/logs/debug-internal.log
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7ff6780ef550>
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_init.py:init():420] calling init triggers
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-03-02 07:49:20,408 INFO    MainThread:18492 [wandb_init.py:init():471] starting backend
2022-03-02 07:49:20,408 INFO    MainThread:18492 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-02 07:49:20,423 INFO    MainThread:18492 [backend.py:ensure_launched():219] starting backend process...
2022-03-02 07:49:20,437 INFO    MainThread:18492 [backend.py:ensure_launched():224] started backend process with pid: 18539
2022-03-02 07:49:20,439 INFO    MainThread:18492 [wandb_init.py:init():480] backend started and connected
2022-03-02 07:49:20,457 INFO    MainThread:18492 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-02 07:49:20,458 INFO    MainThread:18492 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-02 07:49:20,458 INFO    MainThread:18492 [wandb_init.py:init():550] updated telemetry
2022-03-02 07:49:20,487 INFO    MainThread:18492 [wandb_init.py:init():581] communicating current version
2022-03-02 07:49:21,280 INFO    MainThread:18492 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-02 07:49:21,281 INFO    MainThread:18492 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-02 07:49:21,503 INFO    MainThread:18492 [wandb_init.py:init():624] starting run threads in backend
2022-03-02 07:49:24,401 INFO    MainThread:18492 [wandb_run.py:_console_start():1827] atexit reg
2022-03-02 07:49:24,401 INFO    MainThread:18492 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-02 07:49:24,402 INFO    MainThread:18492 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-02 07:49:24,402 INFO    MainThread:18492 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-02 07:49:24,403 INFO    MainThread:18492 [wandb_init.py:init():651] run started, returning control to user process
2022-03-02 07:49:24,405 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:49:24,405 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:49:24,412 INFO    MainThread:18492 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 07:49:24,413 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:49:24,413 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:49:24,418 INFO    MainThread:18492 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 07:49:24,421 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:49:24,422 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:49:24,426 INFO    MainThread:18492 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 07:49:24,428 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:49:24,428 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:49:24,432 INFO    MainThread:18492 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 07:49:27,515 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:49:27,515 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:49:27,520 INFO    MainThread:18492 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 07:49:27,520 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:49:27,520 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:49:27,524 INFO    MainThread:18492 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 07:49:27,530 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:49:27,530 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:49:27,534 INFO    MainThread:18492 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 07:53:15,856 INFO    MainThread:18492 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 07:53:15,856 INFO    MainThread:18492 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 07:56:12,439 INFO    MainThread:18492 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-02 07:56:12,440 INFO    MainThread:18492 [wandb_run.py:_restore():1769] restore
2022-03-02 07:56:13,017 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-03-02 07:56:13,257 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-03-02 07:56:13,576 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 3703
}

2022-03-02 07:56:13,678 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 4029
}

2022-03-02 07:56:13,781 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 4029
}

2022-03-02 07:56:13,883 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 07:56:13,985 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 07:56:14,086 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 07:56:14,188 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 07:56:14,289 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 07:56:14,391 INFO    MainThread:18492 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

